server/views/pages/appointments/create-and-edit/review-prisoners.njk

```diff
@@ -28,21 +28,19 @@
         <div class="govuk-grid-column-two-thirds">
             {{ appointmentTypeCaption(session) }}
             {% if prisoners.length == 0 %}
-                {% if session.appointmentJourney.mode == AppointmentJourneyMode.EDIT %}
+                {% if notFoundPrisoners | length %}
```

Comment:
Reworked to check if `notFoundPrisoners` first if `prisoners.length == 0` as this is consistent across the create, edit and back to back journey

Reply:
Good spot